syntax = "proto3";

package grpc;

message Empty {}

message Message {
    string name = 1;
    string friend = 2;
    bytes message = 3;
}

service MessageServer {
    rpc MessageStream (Empty) returns (stream Message);
    rpc SendMessage (Message) returns (Empty);
}